src/main/java/com/starrocks/connector/flink/table/sink/StarRocksSinkOptions.java

```diff
@@ -359,6 +369,21 @@ public StreamLoadProperties getProperties() {
 
         if (hasColumnMappingProperty()) {
             defaultTablePropertiesBuilder.columns(streamLoadProps.get("columns"));
+        } else if (getTableSchemaFieldNames() != null) {
```

Comment:
Why not add a custom StreamLoadTableProperties for a specific db + table to StreamLoadProperties? I think StreamLoadProperties#tablePropertiesMap is just designed for this case？